File: Backend/Services/Inventory.API/Models/Enums.cs
namespace Inventory.API.Models
{
    public enum Unit
    {
        pcs,
        ml,
        L,
        g,
        kg
    }
    public enum ProductScope
    {
        Global,
        Private,
        Pending
    }
    
    public enum KitchenSortStrategy
    {
        ExpirationDate,
        Quantity,
        Name,
        AddedDate
    }
}

File: Frontend/ReactWeb/src/Components/Ingredients/enums.ts
export const Unit = {
  Pcs: 'pcs',
  Ml: 'ml',
  L: 'L',
  G: 'g',
  Kg: 'kg',
} as const;

export type Unit = typeof Unit[keyof typeof Unit];


export const getUnitLabel = (val: string): string => {
    if (val === Unit.L) return 'L';
    return val; 
};